Library/Formula/rust.rb

```diff
@@ -1,20 +1,27 @@
-require 'formula'
-
 class Rust < Formula
-  homepage 'http://www.rust-lang.org/'
-  url 'https://static.rust-lang.org/dist/rustc-1.0.0-beta-src.tar.gz'
+  homepage "http://www.rust-lang.org/"
   version "1.0.0-beta"
-  sha256 '94248e30487723ac6f6c34a0db5a21085c0b1338e6a32bd12b159e1d2cd80451'
 
-  head 'https://github.com/rust-lang/rust.git'
+  stable do
+    url "https://static.rust-lang.org/dist/rustc-1.0.0-beta-src.tar.gz"
+    sha256 "94248e30487723ac6f6c34a0db5a21085c0b1338e6a32bd12b159e1d2cd80451"
+    resource "cargo" do
+      url "https://github.com/rust-lang/cargo.git", :revision => "84d6d2c"
```

Comment:
Sadly, no. As has been mentioned [previously](https://github.com/Homebrew/homebrew/issues/33322#issuecomment-92890011) and summarised above, time of rustc release defines the cargo nightly that gets pulled in the rust release. I've been [prodding upstream](https://github.com/rust-lang/cargo/issues/1178#issuecomment-92886647) to have them at least tag the corresponding SHA1 (and optimally create a GitHub release) but interest, albeit sympathetic, appears somewhat limited.